Utility CSS/utility.css

```diff
@@ -100,3 +100,6 @@
 .margin-m1 {
   margin: 4px 6px;
 }
+.margin-m2 {
+  margin: 8px 10px;
+}
```

Comment:
Try to use relative units here.

Reply:
hey as vikram said below 10px you should not use rem so i used px